cep-XXXX.md

```diff
@@ -24,21 +24,37 @@ The `environment.yml` file format was introduced by `conda env`. It has been tra
 
 ## Specification
 
-`environment.yml` files are YAML documents that encode the necessary information to create a new conda environment. They can contain up to seven top-level keys:
+`environment.yml` files are YAML documents that encode the necessary information to create a new conda environment. The core standard recognizes five top-level keys:
 
 - `name`: The preferred name for the environment.
 - `prefix`: The preferred full path to the environment. Often ignored.
 - `dependencies`: List of package specifications that MUST be installed in the new environment.
 - `channels`: conda channels that will be used to resolve the dependencies.
 - `variables`: Environment variables that SHOULD be added to the `conda-meta/state` file in the resulting environment.
+
+The file name is not standardized, but the extension MUST be `yml` or `yaml`.
+
+### Optional extensions
+
+A number of extensions have been implemented by some conda clients. These are not guaranteed
+to be available in all conda clients.
+
+Two additional top-level keys MAY be present:
+
 - `platforms`: Target subdirs for the solver.
 - `category`: Free-form field to label the dependencies as a group.
 
-Additional sections can be present, but they MUST be ignored and the user SHOULD receive an informative warning about them.
+Preprocessing selectors MAY be available to specify line filters to allow platform-specific values,
+among other uses. There are two types: comment-based and dictionary-based. Only one type of filter
+SHOULD be used in the same document.
 
-The file name is not standardized, but the extension MUST be `yml` or `yaml`.
+### Other top-level keys
+
+Additional top-level keys can be present, but they MUST be ignored and the user SHOULD receive an informative warning about them when the conda client does not support them.
```

Comment:
Do we mean to explicitly forbid any more extensions to the v0 format?

Reply:
If they are, they will be non-standard (as the currently listed extensions "were"). I really hope no one needs more from this til-now unspecified format.